src/lib/mina.ts

```diff
@@ -369,6 +372,11 @@ interface Mina {
 }
 
 const defaultAccountCreationFee = 1_000_000_000;
+const defaultNetworkConstants = {
+  genesisTimestamp: UInt64.from(0),
+  slotTime: UInt64.from(3 * 60 * 1000),
+  accountCreationFee: UInt64.from(defaultAccountCreationFee),
```

Comment:
do we need the account creation fee twice or can we simplify to only use one source?

Reply:
Maybe then we should:
1. Remove [accountCreationFee API](https://github.com/o1-labs/o1js/pull/1367/files#diff-1f92503aa572cd2ea7bc843e4e77809f3aef81f49bb25995a80d032127482ffcR749);
2. And allow return [default network constants](https://github.com/o1-labs/o1js/pull/1367/files#diff-1f92503aa572cd2ea7bc843e4e77809f3aef81f49bb25995a80d032127482ffcR375) instead of [throwing](https://github.com/o1-labs/o1js/pull/1367/files#diff-1f92503aa572cd2ea7bc843e4e77809f3aef81f49bb25995a80d032127482ffcR768);
3. Update all API usage (obviously).

WDYT?

Reply:
sounds good, but please deprecate the old API, not remove